lib/util.ts

```diff
@@ -0,0 +1,168 @@
+import axios from 'axios';
```

Comment:
I don't like calling this util and it's not a pattern we should copy. Let's call it soapRequest or something. This file should also be chopped up into smaller modules and not one big file.